components/hab/src/main.rs

```diff
@@ -161,13 +160,11 @@ fn start(ui: &mut UI) -> Result<()> {
 }
 
 fn sub_cli_setup(ui: &mut UI) -> Result<()> {
-    let fs_root = henv::var(FS_ROOT_ENVVAR).unwrap_or(FS_ROOT_PATH.to_string());
-    let fs_root_path = Some(Path::new(&fs_root));
```

Comment:
We'll still need these `FS_ROOT_ENVVAR` checks here, for non-Windows Studio and `hab pkg export` tooling. The check used in `core` only applies to Windows when potentially working in the Studio--I can fill you in more if that's helpful, but we'll want to preserve these checks in `main.rs`.